website/blog/2023-05-25-pact-open-source-update-may-2023.md

```diff
@@ -0,0 +1,123 @@
+---
+
+It's May, the sunshine ☀️ is upon us, and you might be reading this in the park 🏞️. If you are, watch out, theres a cyclist 🚴coming. 🛎️ 🛎️
+
+Phew, that was close. Right, well now we are sorted, we've got some goodies lined up for this month, as the city of open-source never sleeps.
+
+## OSS Updates
+
+### Pact-Net
+
+V4 of `Pact-Net` is about to support `V4` of the `Pact-Specification`.
+  - https://github.com/pact-foundation/pact-net/pull/453
+
+The OCD gods have been appeased. You can all rest easy now, thanks to Adam Rodgers.
+
+### Pact-PHP
+
+- `pact-php` `ffi-dev` branch
+  - Tien Vo has been working hard on `ffi` support in `pact-php` and we are grateful to have had a couple more reviewers join the party. We've wanted to make it easier to consume, for beta testing, in order to get feedback from users, who may be more familiar in their own codebase, than `pact-php`'s. You can read more details about it in this post, on how you can get involved.
+- `pact-php` with the `ruby` reference core
+  - Introduces Arm64 Linux / MacOS
+
+### Pact-Python
+
+- `pact-python` has put out a deprecation notice, noting that CI testing will soon be switched off for Python `3.6`.
+  - You can check the [Python EOL dates](https://endoflife.date/python) and get prepared. `3.7` will be dropping support shortly, so you might as well take the time to jump up to `3.11` whilst you are there.
+- `pact-python` with the `ruby` reference core
+  - Introduces Arm64 Linux / MacOS
+  - Reintroduces x86 Windows
+
+### Pact-JS
+
+- `pact-js` & `pact-js-core` has put out a deprecation notice, noting that CI testing will soon be switched off for Node `14`.
+  - You can check the [Node EOL dates](https://endoflife.date/node) and get prepared. `16` will be dropping support shortly, so you might as well take the time to jump up to `18` which is the latest `LTS` but also check that you are compatible with `20`, the upcoming `LTS`
+- `pact-js-core` with the `ruby` standalone binaries
+  - Introduces Arm64 Linux / MacOS
+  - Closes currents issues for users on Linux arm64 machines who cannot install `pact-js-core` or `pact-js` despite us using the `rust` reference core.
+- `pact-js-core` will soon come will `batteries included`
+  - All supported platform/arch combo's for `libpact_ffi` will come pre-packaged in `pact-js-core` and `pact-js`, this means users will be able to
+  - `npm install @pact-foundation/pact --ignore-scripts`
+  - Not require a full developer tool-chain included Python to run `node-gyp` in order to generate the required `node` bindings. We'll do the grunt work, so you don't have to.
+
+### Pact-Broker
+
+As the `pact_broker` nears its birthday, it decided that is better get ready. You'll now find the latest `pact_broker` and `pact_broker-docker` projects have been upgrades to support Ruby `3.2`.
+
+You'll be able to pull multi-arch builds for `arm/arm64` machines soon, by adding the `-multi` flag to your `pact_broker` tag.
+
+### Pact FFI
```

Comment:
Lots of activity in https://github.com/pact-foundation/pact-reference/blob/master/rust/pact_ffi/CHANGELOG.md#044---bugfix-release.